### ISSUE TITLE
ALOps Extension API can't GET the list of companies (wrong API Version)

### ISSUE THREAD

**Issue by mbug** [HodorNV/ALOps#337]
**Describe the bug**
ALOps Extension API fails to Get the list of companies. I'm quite certain the problem is a wrong API version. It is trying to use v1.0 instead of beta. I'm trying to release an App on BC 13.4.

**the used yaml**
please provide the yaml that you used.  It helps you put the yaml like this: 
```yaml
#Your build pipeline references an undefined variable named ‘Username’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab. See https://go.microsoft.com/fwlink/?linkid=865972
#Your build pipeline references an undefined variable named ‘Password’. Create or edit the build pipeline for this YAML file, define the variable on the Variables tab. See https://go.microsoft.com/fwlink/?linkid=865972
variables:
  Endpoint: 'https://swarm-xzmuxlla.westeurope.cloudapp.azure.com/3284660b2a2arest/api'

steps:
- task: Hodor.hodor-alops.ALOpsExtensionAPI.ALOpsExtensionAPI@1
  displayName: 'ALOps Extension API'
  inputs:
    interaction: publish
    api_endpoint: '$(Endpoint)'
    authentication: basic
    username: '$(Username)'
    password: '$(Password)'
```

**the output**
Also the complete output is necessary for us to see what is going on.  Also use backtics:
```
2021-03-05T22:56:25.4914366Z ----                                                                                        -----
2021-03-05T22:56:25.4916610Z usedocker                                                                                   False
2021-03-05T22:56:25.4918548Z fixed_tag                                                                                        
2021-03-05T22:56:25.4920588Z interaction                                                                               publish
2021-03-05T22:56:25.4922612Z api_endpoint            https://swarm-xzmuxlla.westeurope.cloudapp.azure.com/3284660b2a2arest/api
2021-03-05T22:56:25.4924900Z authentication                                                                              basic
2021-03-05T22:56:25.4927003Z azure_tenant_id                                                                                  
2021-03-05T22:56:25.4928787Z azure_app_client_id                                                                   $(ClientId)
2021-03-05T22:56:25.4931640Z azure_app_client_secret                                                           $(ClientSecret)
2021-03-05T22:56:25.4935198Z username                                                                              MBUGIEL6044
2021-03-05T22:56:25.4937475Z password                                             ***
2021-03-05T22:56:25.4939291Z bccompany                                                                                        
2021-03-05T22:56:25.4941234Z artifact_path                                                                                    
2021-03-05T22:56:25.4943035Z artifact_filter                                                                             *.app
2021-03-05T22:56:25.4944814Z showdeploymentstatus                                                                         True
2021-03-05T22:56:25.4975979Z 
2021-03-05T22:56:25.4981485Z 
2021-03-05T22:56:25.5088770Z 
2021-03-05T22:56:25.5171440Z *** For documentation, please visit   : https://www.alops.be/documentation
2021-03-05T22:56:25.5172533Z 
2021-03-05T22:56:26.0315592Z *** ALOps License:
2021-03-05T22:56:26.0335742Z   * Licensed To: Cosmo Consult SSC GmbH
2021-03-05T22:56:26.0347230Z 
2021-03-05T22:56:26.8991461Z *** Imported required PS-Functions
2021-03-05T22:56:26.9004786Z *** Parse Task Parameters
2021-03-05T22:56:26.9263647Z *** Creating API Connector with endpoint [https://swarm-xzmuxlla.westeurope.cloudapp.azure.com/3284660b2a2arest/api]
2021-03-05T22:56:26.9444993Z *** Interaction Mode: publish
2021-03-05T22:56:26.9477581Z *** API Authentication: Using Basic Authentication [MBUGIEL6044]
2021-03-05T22:56:26.9546945Z *** List Companies
2021-03-05T22:56:26.9648564Z ##[command]Invoke-RestMethod -Method Get -Uri 'https://swarm-xzmuxlla.westeurope.cloudapp.azure.com/3284660b2a2arest/api/microsoft/automation/v1.0/companies'
2021-03-05T22:56:27.2236172Z ##[error]Exception in BCConnector.GetAPIData: The remote server returned an error: (404) Not Found.
2021-03-05T22:56:27.2288777Z *** Selected Company:  []
2021-03-05T22:56:27.4980323Z ##[error]Cannot find an overload for "GetExtensions" and the argument count: "1".
2021-03-05T22:56:27.6578471Z ##[section]Finishing: ALOps Extension API
```

**Expected behavior**
Instead of:
2021-03-05T22:56:26.9648564Z ##[command]Invoke-RestMethod -Method Get -Uri 'https://swarm-xzmuxlla.westeurope.cloudapp.azure.com/3284660b2a2arest/api/microsoft/automation/**v1.0**/companies'

I would expect:
2021-03-05T22:56:26.9648564Z ##[command]Invoke-RestMethod -Method Get -Uri 'https://swarm-xzmuxlla.westeurope.cloudapp.azure.com/3284660b2a2arest/api/microsoft/automation/**beta**/companies'

**Screenshots**
![image](https://user-images.githubusercontent.com/42273207/110185725-69d55880-7e13-11eb-8f32-22b6e93e3f44.png)

**Additional context**
Is there maybe already a possibility to define the API Version?

**Comment by waldo1001** [comment]
Hm, that's an old version - we'll try to make it configurable.. .

**Comment by AdminHodor** [comment]
Dear @mbug ,

We just released an intermediate build ALOps v1.440.2365 which should resolve this issue.
A new parameter "apiversion" is added to the ALOpsExtensionAPI task which default to "v1.0", for your case you should set this parameter to "beta".

Please note it could take a couple of pipeline runs before DevOps picks up the new version.

Kind regards,

**Comment by mbug** [comment]
Thank you so much!
